app/templates/components/filter-by.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if isShowingBody}}
+  <div class="filter-table">
```

Comment:
User bs-dropdown and clean up this component.
Use this teamList inside the bsdropdown menu property

Reply:
I have fixed this change in next PR.